crates/sui-framework/docs/bridge/message.md

```diff
@@ -814,7 +814,7 @@ Token Transfer Message Format:
     <a href="../bridge/message.md#bridge_message_payload">payload</a>.push_back((vector::length(&target_address) <b>as</b> u8));
     <a href="../bridge/message.md#bridge_message_payload">payload</a>.append(target_address);
     <a href="../bridge/message.md#bridge_message_payload">payload</a>.push_back(<a href="../bridge/message.md#bridge_message_token_type">token_type</a>);
-    // bcs serialzies u64 <b>as</b> 8 bytes
+    // bcs serializes u64 <b>as</b> 8 bytes
```

Comment:
You probably already know this but these files (`crates/sui-framework/docs/**/*.md`) are created automagically so editing them manually will break the build. I'm betting you created these through cargo after updating the source files but leaving this here before I forget jic.